lib/bolt_server/transport_app.rb

```diff
@@ -514,10 +514,10 @@ def make_winrm_target(target_hash)
 
     # Fetches the metadata for a single plan
     #
-    # @param project_ref [String] the project to fetch the plan from
+    # @param versioned_project [String] the project to fetch the plan from
     get '/project_plans/:module_name/:plan_name' do
-      return MISSING_PROJECT_REF_RESPONSE if params['project_ref'].nil?
-      in_bolt_project(params['project_ref']) do |context|
+      return MISSING_PROJECT_REF_RESPONSE if params['versioned_project'].nil?
```

Comment:
Nit: we could probably change this to `MISSING_VERSIONED_PROJECT_RESPONSE` to be a little clearer

But definitely don't block on that :)

Reply:
Yep, good call, let me change that

Reply:
Updated